.github/workflows/black-push.yml

```diff
@@ -0,0 +1,26 @@
+# Taken from https://github.com/cclauss/autoblack
+# See https://github.com/cclauss/autoblack/blob/master/.github/workflows/autoblack_push.yml
+
+name: autoblack-push
+on:
+  push:
+    branches: [ main ]
```

Comment:
Maybe worth responding to all pushes to catch any possible issues before they hit the main branch?
```suggestion

```

Reply:
I've been using `on: [push, pull_request]` personally to also run on pull requests. That would also be a good option

```diff
-on:
-  push:
-    branches: [ main ]
+ on: [push, pull_request]

Reply:
Will leave PRs to be formatted on merge, though they should be pre-formatted by a workflow on the branch anyway.